### DISCUSSION TITLE
Where is my admin nprofile?

### DISCUSSION THREAD

**Discussion by G12c4** [shocknet/wallet2#318] [Q&A] [answered]
Hello beautiful pub team, I deployed the pub server and successfully imported the admin profile to ShockWallet. However, after updating to the latest APK version on my phone, I now have two apps. Unfortunately, I didn’t save the admin profile.

To retrieve it, I found two files on the server: admin.npub and app.nprofile. Unfortunately, the admin.npub file does not work, and the app.nprofile gives me an account with zero balance.

**Comment by shocknet-justin** [comment, marked as answer]
Hi, sorry about the trouble, we've been getting the android builds ready for the play store and that caused the last release to ID differently.

Two options, 

You can auth into the sync service on the first app, and then again in the second, so that the admin secret syncs across.

Other option is you can delete the admin.npub file from the node, that will cause a new file to be created automatically, `.admin_connect`, which will contain a refreshed admin secret you can add to wallet. (This resets the admin key, IIRC the balance move would be manual)

**Reply by G12c4** [reply]
Neither option worked for me. Sanctum wouldn’t let me sign in with my existing Nostr profile. I tried using three different browsers on my phone. I also deleted the admin.npub file, but no new file was created. I searched for the file in all other directories and even tried restarting the service, but nothing helped. Unfortunately, I had to transfer my funds to another wallet, delete the pub and LND services, and then delete the directory to install a new instance.

**Reply by shocknet-justin** [reply]
Ugh, apologies... that's one way to do it anyway 😅

Will have a fix for nsec sanctum auth shortly, its been primarily a solution for email users but we're expanding it tol add a self-hosted remote signers

The `.admin_connect` file since leading with a . may have been hidden from view by default, from a cli `ls -alh` should have shown it.

**Reply by G12c4** [reply]
Oh totally missed the dot 🤦🏻‍♂️
